### PR TITLE
zh-translation: docs/root/configuration/listeners/network_filters/thr…

### PR DESCRIPTION
**Pull request by vgbhfive**
For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:  docs/root/configuration/listeners/network_filters/thrift_proxy_filter.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #327
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/configuration/listeners/network_filters/thrift_proxy_filter.rst [hunk 1]
```diff
@@ -1,62 +1,45 @@
...
-------------------------
+集群协议选项
+--------------
 
-Thrift connections to upstream hosts can be configured by adding an entry to the appropriate
-Cluster's :ref:`extension_protocol_options<envoy_v3_api_field_config.cluster.v3.Cluster.typed_extension_protocol_options>`
-keyed by `envoy.filters.network.thrift_proxy`. The
-:ref:`ThriftProtocolOptions<envoy_v3_api_msg_extensions.filters.network.thrift_proxy.v3.ThriftProtocolOptions>`
-message describes the available options.
+到上游主机的 Thrift 连接可以通过向适当的集群中的 :ref:`extension_protocol_options<envoy_v3_api_field_config.cluster.v3.Cluster.typed_extension_protocol_options>` 添加一个由 `envoy.filters.network.thrift_proxy` 作为键的键值对来配置。
```
Comment (addozhang) [inline docs/root/configuration/listeners/network_filters/thrift_proxy_filter.rst]:
```suggestion
到上游主机的 Thrift 连接可以通过向相关的集群中的 :ref:`extension_protocol_options<envoy_v3_api_field_config.cluster.v3.Cluster.typed_extension_protocol_options>` 添加一个由 `envoy.filters.network.thrift_proxy` 作为键的记录来配置。
```

File: docs/root/configuration/listeners/network_filters/thrift_proxy_filter.rst [hunk 2]
```diff
@@ -1,62 +1,45 @@
...
+~~~~~~~~~~~~~~~~
 
-Request metadata that is available for routing (see above) is automatically converted between wire
-formats when translation between downstream and upstream connections occurs. Twitter protocol
-request contexts, client id, destination, and delegations are therefore presented as Header
-transport key/value pairs, named as above. Similarly, Header transport key/value pairs are
-presented as Twitter protocol RequestContext values, unless they match the special names described
-above. For instance, a downstream Header transport request with the info key ":client-id" is
-translated to an upstream Twitter protocol request with a ClientId value.
+当下游和上游连接之间发生转换时，可用于路由的请求元数据（见上下文）将在连接格式之间自动转换。Twitter 协议请求上下文、客户端标识、目的地和 delegations 被表示如上所述的为 Header 传输键值对。
```
Comment (addozhang) [inline docs/root/configuration/listeners/network_filters/thrift_proxy_filter.rst]: delegations 是不是也要翻译一下？

File: docs/root/configuration/listeners/network_filters/thrift_proxy_filter.rst [hunk 3]
```diff
@@ -1,62 +1,45 @@
...
+类似地，Header 传输键值对可以被表示为 Twitter 协议请求上下文，除非他们与上面描述的特殊名称匹配。例如，带有信息键 ":client-id" 的下游 Header 传输请求被转换为具有 ClientId 值的上游 Twitter 协议请求。
```
Comment (addozhang) [inline docs/root/configuration/listeners/network_filters/thrift_proxy_filter.rst]: 双引号要使用中文的

**Review by addozhang — COMMENTED** [review]
@majinghe 文中的 header 是否要翻译？

**Comment by majinghe** [comment]
@addozhang 晓辉。header 需要翻译。前期都翻译成头部或者头了。delegation 这个不需要翻译，昨晚上在群里讨论了。

**Review by addozhang — APPROVED** [review]
/LGTM

**Pull request merged**